test/sdk/restapi/swagger/model_stream_result_of_sdk_game_server.go

```diff
@@ -12,13 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// This code was autogenerated. Do not edit directly.
 /*
- Hand crafted file, due to bugs in swagger-gen-cli
-*/
-
+ * sdk.proto
+ *
+ * No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)
+ *
+ * API version: version not set
+ * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
+ */
 package swagger
 
-type XStreamDefinitionssdkGameServer struct {
-	Result *SdkGameServer `json:"result"`
-	Error *RuntimeStreamError `json:"error"`
-}
\ No newline at end of file
+type StreamResultOfSdkGameServer struct {
+	Result *SdkGameServer `json:"result,omitempty"`
+	Error_ *GooglerpcStatus `json:"error,omitempty"`
```

Comment:
Why does the Error field need to be underlined?

Reply:
This is by Swagger Codegen auto-generation.